fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/api/client/keybinding/FabricKeyBinding.java

```diff
@@ -28,8 +32,24 @@
  * {@link KeyBindingRegistry#register(FabricKeyBinding)}!
  */
 public class FabricKeyBinding extends KeyBinding {
+
+	private final Identifier id;
+
 	protected FabricKeyBinding(Identifier id, InputUtil.Type type, int code, String category) {
-		super("key." + id.toString().replace(':', '.'), type, code, category);
+		super(formatKeyName(id), type, code, category);
+		this.id = id;
+	}
+
+	public static String formatKeyName(final Identifier id) {
```

Comment:
Why is this the only place where the parameter is final ?

Reply:
Presumably because `id`. I'll remove the final.